resolveImports.mjs

```diff
@@ -1,7 +1,19 @@
+// @ts-check
+
 import { parse } from "es-module-lexer";
 import path from "node:path";
 import { readFile, access } from "node:fs/promises";
 
+/**
+ * Resolved imports.
+ * @typedef {string[] | undefined} ResolvedImports
```

Comment:
```suggestion
 * @typedef {Array<string> | undefined} ResolvedImports
```

The `string[]` is a valid type, but I much prefer the `Array<string>` syntax because it's consistent with every other generic type, and also it's nicer to read `Array` first and know it's a list of things, than read the name of the item type first, then be surprised by later (or not noticing) the `[]` after it. When it's a small type like this it's less important, but when you have big inline object types on multiple lines, you have to scroll down the code to see if it's in an array or not.

Reply:
I must have overlooked this one as I had corrected the others. Good points.